charts/fission-all/config/fluentbit.conf

```diff
@@ -0,0 +1,56 @@
+# Some stuff in this file is copied out of https://github.com/fluent/fluent-bit-kubernetes-logging/blob/master/output/kafka/fluent-bit-configmap.yaml
+[SERVICE]
+    Flush 5
+    Log_Level info
+    Parsers_File parsers.conf
+
+[INPUT]
+    Name tail
+    Tag kube.*
+    Path ${LOG_PATH}
+    Mem_Buf_Limit 5MB
+    Parser docker
+    DB /tmp/flb_kube.db
```

Comment:
The `DB` is used to keep track of monitored files and offsets. If we don't keep this file, the log will be doubled if the fluentbit pod is restarted. A solution is to keep this file in node's filesystem by mounting a hostPath volume.

Reply:
good catch!